README.md

```
@@ -185,9 +200,11 @@ A set of functions for the **F**riendly **I**nteractive **Sh**ell (fish). This i
   - **`taskd`**: set the task with the given id as done;
   - **`taskflush`**: set all expired tasks to done;
   - **`teeth`**: help cleaning teeth;
+  - **`timeprompt`**: wait for a given amount of time, or until the person hits a key exits with 1 if the user interrupted;
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)